BookApi/BookApi/Service/BookService.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using BookApi.Models;
```

Comment:
```suggestion
using BookApi.Models;
```
VSCode shows these as unused imports :)